src/Libraries/libSystem/libsystem_kernel/CMakeLists.txt

```diff
@@ -234,15 +400,64 @@ target_sources(libsystem_kernel PRIVATE
     wrappers/string/strlcpy.c
     wrappers/string/strlen.c
     wrappers/string/strsep.c
+    wrappers/_libkernel_init.c
+    wrappers/_libc_funcptr.c
+    wrappers/system-version-compat.c
 )
-target_include_directories(libsystem_kernel PUBLIC include)
-target_include_directories(libsystem_kernel PUBLIC ${CMAKE_CURRENT_BINARY_DIR}/public_include)
-target_include_directories(libsystem_kernel PRIVATE bootstrap_include wrappers/string)
-target_compile_definitions(libsystem_kernel PRIVATE _FORTIFY_SOURCE=0)
+target_include_directories(libsystem_kernel PUBLIC include ${CMAKE_CURRENT_BINARY_DIR}/public_include)
+target_include_directories(libsystem_kernel PRIVATE wrappers/string)
+
+# Copied from static target above
+target_include_directories(libsystem_kernel PRIVATE mach mach/mach mach/servers custom wrappers ${CMAKE_CURRENT_BINARY_DIR}/internal_include)
+target_include_directories(libsystem_kernel PRIVATE libsystem_kernel_private_headers)
+
+target_compile_options(libsystem_kernel PRIVATE -fdollars-in-identifiers -no-cpp-precomp -fno-common -fno-stack-protector -fno-stack-check -momit-leaf-frame-pointer -Wno-nullability-completeness -Wno-implicit-function-declaration)
+target_compile_definitions(libsystem_kernel PRIVATE _FORTIFY_SOURCE=0 __DARWIN_VERS_1050=1 NO_SYSCALL_LEGACY SYSCALL_PRE1050)
+
 target_link_libraries(libsystem_kernel PRIVATE libsystem_kernel_static)
-target_link_options(libsystem_kernel PRIVATE -umbrella System -all_load -nostdlib)
-install(TARGETS libsystem_kernel DESTINATION /usr/lib/system COMPONENT BaseSystem)
+target_link_libraries(libsystem_kernel PRIVATE CrashReporterClient)
+target_link_options(libsystem_kernel PRIVATE -umbrella System -all_load)
```

Comment:
`-nostdlib` needs to be put back.

Reply:
Closing resolved stale conversation. (GitHub doesn’t call it as such, but that’s what it is.)